docs/integrations/sources/referralhero.md

```diff
@@ -0,0 +1,28 @@
+# referralhero
+airbyte connector developed using airbyte 1.0 UI Connector development
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `uuid` | `string` | uuid.  |  |
```

Comment:
Please check this Loom video why you shouldn't use this config parameter and must implement a parent stream based on lists.

Reply:
[Loom](https://www.loom.com/share/9e5d54e80f76440cb4ebcc65242059e3?sid=5ef0bed9-ae11-40fa-9ca6-a3e19eef432e)

Reply:
implemented parent stream based on the list, thank you